### DISCUSSION TITLE
Email Id Problem

### DISCUSSION THREAD

**Discussion by Rishikr15** [community/community#60534] [GitHub Education]
### Hi everyone,
The following is the email i received from Github Education

-------------------------------------------
Hey Rishi Kumar,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jun 26, 2023 for [@Rishikr15](https://github.com/Rishikr15).

**_**Attendees of this school must verify and use their academic email address.**_**

Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/).

Have questions? Join the [GitHub Education Community](https://github.com/orgs/community/discussions/categories/github-education) for answers.

💖 The GitHub Education Team"""""""""""

--------------------------------------
**Query:--
How the Attendees verify the application
What should be the contents of the email Id
What is the email id where the attendees will send their verification emails to?**

![github_education_email](https://github.com/community/community/assets/91412485/f98a3ee6-17c8-4c04-a2dd-4fa8fd3ecf2e)

**Comment by samus-aran** [comment]
Hi @Rishikr15 

It's stating that your selected school in the application requires students to have a verified academic email supplied by your school. 

Hope that helps 👍

**Reply by Rishikr15** [reply]
1. But i didn't get a verified email by my school but it didn't create any issue earlier for verification purpose as i gave my official email id in 2021 and 2022 (rishi61kumar@gmail.com) but now it is hindering my application 
2. I have given my college fee receipt for this year and my college id card still it is not verifying the application.
3. Second issue in case my College mentor provides his/her email id and gives an explanation on this issue from the official college provided email id (<name>.gnit.ac.in) ,then what kind of contents should be given in the email so as to verify my application.
Thank you

**Reply by ajeetkumarrauniyar** [reply]
> Hi @Rishikr15
> 
> It's stating that your selected school in the application requires students to have a verified academic email supplied by your school.
> 
> Hope that helps 👍

I have a verified email ID from the University `O23BCA110517@cuchd.in` Then why it is ? Any Idea
